client/src/main/java/io/pravega/client/ClientFactory.java

```diff
@@ -11,6 +11,7 @@
 
 import com.google.common.annotations.Beta;
 import io.pravega.client.batch.BatchClient;
+import io.pravega.client.byteStream.ByteStreamClient;
```

Comment:
If we are separating out the byte stream client in its own package, should we do the same for the event stream client?

Reply:
Yes, we should. I wasn't planning to do that in this PR, because for one thing, that is a API breaking change.

Reply:
Ok, I created issue #3007 for this.